apps/transport/priv/gettext/en/LC_MESSAGES/validations.po

```diff
@@ -316,3 +316,11 @@ msgstr ""
 #, elixir-autogen, elixir-format
 msgid "from"
 msgstr ""
+
+#, elixir-autogen, elixir-format
+msgid "doc-eu-formats"
+msgstr "To access informations related to european standards, you can browse <a href=\"%{link}\" target=\"_blank\">our documentation</a>."
```

Comment:
```suggestion
msgstr "To access information related to European standards, you can browse <a href=\"%{link}\" target=\"_blank\">our documentation</a>."
```

information ne peut pas être mis au [pluriel](https://jakubmarian.com/information-is-vs-informations-are-is-information-plural-or-singular-in-english/). Je pense que European se met avec une majuscule.

Reply:
Merci !